src/DebugEventDispatcher.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Yiisoft\Yii\Debug;
+
+use Psr\EventDispatcher\EventDispatcherInterface;
+
+interface DebugEventDispatcher extends EventDispatcherInterface
```

Comment:
Is there going to be more than a single implementation of it?

Reply:
Unlikely. At now it used to late binding in https://github.com/yiisoft/yii-debug/blob/5fed4ba88a80b6b3428783bca9044ff54fc4e5be/src/DebugServiceProvider.php#L34

Reply:
How about removing the interface then and using class name directly then?

Reply:
I think no, because it may redeclare `EventDispatcherInterface`

Reply:
You can get an instance of concrete class from container as well.

Reply:
It can redeclare the `CompositeDispatcher` too :C